### PR TITLE
feature/artwork-article

### PR DESCRIPTION
**Pull request by coding-tomato**
## Trello ticket
[Ticket Link](https://trello.com/c/5rB7BjTC)

## Description
Artwork article component to showcase different artworks. It's still missing the URL for the "More information button", this will be added in a bugfix PR.

## To reproduce
1. Run the project
2. Open the specific page (e.g. http://localhost:8080/obras/[decirAdios])
3. Check the component

## Screenshots
Mobile
image
image

Desktop
image

### CODE REVIEW

**Review by dyepes62 — APPROVED** [review]

File: components/ArtworkArticle/ArtworkDetails.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export default function ArtworkDetails({ data }) {
+  const { dimensions, technique, price } = data;
+  return (
+    <section className="artwork-details__details-container">
+      <p>Técnica:</p>
+      <p>{dimensions}</p>
+      <p>Dimensiones:</p>
+      <p>{technique}</p>
```
Comment (SinuheM) [inline components/ArtworkArticle/ArtworkDetails.js:6]: Here the values are inverted, technique and dimensions

File: styles/components/_artworkArticleDetails.scss [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    height: 1px;
+    width: 100%;
+  }
+}
+
+@media screen and (min-width: $tablet-viewport + 1) {
+  .artwork-details {
+    &__container {
+      max-width: 448px;
+      margin-left: 80px;
```
Comment (SinuheM) [inline styles/components/_artworkArticleDetails.scss]: I think there are margin to both sides

image

Comment (SinuheM) [reply styles/components/_artworkArticleDetails.scss]: Also the font size is a little small 😅

Comment (coding-tomato) [reply styles/components/_artworkArticleDetails.scss]: Might I be using the wrong font size? I'm just using body2, as stated in the styleguide and Figma document.

File: components/ArtworkArticle/ArtworkDetails.js (hunk 1 above)
Comment (coding-tomato) [reply components/ArtworkArticle/ArtworkDetails.js:6]: Wow! Nice catch lol thank you!

**Review by SinuheM — APPROVED** [review]

**Pull request merged**